service.kn.switchtimer/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+<addon id="service.kn.switchtimer" name="KN Switchtimer Service" provider-name="Birger Jesch" version="1.2.4">
+    <requires>
+        <import addon="xbmc.addon" version="16.0.0"/>
+        <import addon="xbmc.python" version="2.24.0"/>
+        <import addon="xbmc.json" version="6.32.4"/>
+    </requires>
+    <extension point="xbmc.service" library="service.py" start="login" />
+    <extension point="xbmc.python.script" library="handler.py" />
+    <extension point="kodi.context.item">
+        <menu id="kodi.core.main">
+            <item library="addtimer.py">
+                <label>30040</label>
+                <visible>System.HasAddon(service.kn.switchtimer) + Window.IsVisible(tvguide)</visible>
+            </item>
+            <item library="deltimer.py">
+                <label>30041</label>
```

Comment:
The context menu entry always disappears when add-on is de-activated. No need to check for that visible condition.